### DISCUSSION TITLE
Project 4 Help

### DISCUSSION THREAD

**Discussion by rachael-clark** [IBAHCM/RPiR#474] [Archive] [answered]
Hello, 

I was wondering if I could get some help with Project 4 practical please?

If I understand the task, I have to make a new diversity profile function that has a vector of qs as an argument, and it calls in the general diversity function we made previously for the function to loop through. 

I thought I had created a suitable for loop with if-else statements to add a vector sequence together, but I can't seem to get it to work, and keep coming up with errors :( really not sure where I am going wrong!

**Comment by rachael-clark** [comment, marked as answer]
Crisis averted ! Bracket in the wrong place... :')

**Reply by richardreeve** [reply]
Hi @rachael-clark - sorry for taking so long to get back to you here - glad you fixed it yourself though!
